in_text/text_03_config.yml

```
@@ -148,7 +148,7 @@ entities:
     -
       attr-label: H
       attr-def: >-
-        Comma-separated list of lake elevations
+        Comma-separated list of lake elevations 
```

lake areas for which lakes? is there a companion to this that explains what these represent?
```suggestion
        Comma-separated list of lake areas
```

---

Updated via this commit https://github.com/USGS-R/forecast-data-release-v2/pull/13/commits/1330ea127ec2c677d95b1cdebcc66f20f54b3190